src/NeuralNetDiffEq.jl

```diff
@@ -4,24 +4,25 @@ using Reexport, Statistics
 @reexport using DiffEqBase
 
 using Flux, Zygote, DiffEqSensitivity, ForwardDiff, Random, Distributions
-using DiffEqFlux, Adapt, CuArrays
+using DiffEqFlux, Adapt, CuArrays, StochasticDiffEq
 import Tracker, Optim
 
 abstract type NeuralNetDiffEqAlgorithm <: DiffEqBase.AbstractODEAlgorithm end
 
-struct TerminalPDEProblem{G,F,Mu,Sigma,X,T,P} <: DiffEqBase.DEProblem
+struct TerminalPDEProblem{G,F,Mu,Sigma,X,T,P,K} <: DiffEqBase.DEProblem
     g::G
     f::F
     μ::Mu
     σ::Sigma
     X0::X
     tspan::Tuple{T,T}
     p::P
```

Comment:
make a section for `A` in here and then have the constructor specifically have a kwarg for `A`.